dask/dataframe/core.py

```diff
@@ -5127,6 +5129,10 @@ def set_index(
         inplace: bool, optional
             Modifying the DataFrame in place is not supported by Dask.
             Defaults to False.
+        sort: bool, optional
+            If True, sort the dataframe by the new index. Otherwise
+            set the index on the individual existing partitions.
+            Defaults to True.
```

Comment:
Nit

```suggestion
            If ``True``, sort the DataFrame by the new index. Otherwise
            set the index on the individual existing partitions.
            Defaults to ``True``.
```